src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const regDiscount = 20;
```

Comment:
Don't try to shorten variable names if unnecessary
```suggestion
  const regularDiscount = 20;
```